config/config.toml.example

```diff
@@ -31,6 +31,12 @@ split-table = true
 # The limit of concurrent executed sessions.
 token-limit = 1000
 
+# store-limit is used to restrain TiDB from sending request to some stores which is up to the limit.
+# It is useful when some stores are broken and block any request sent to it, which will finally
+# cause all token limits are occupied by this store, consequently leading QPS to decrease sharply.
+# [token-limit/stores, token-limit/2] recommended, 0 means shutting off store limit.
+store-limit = 500
```

Comment:
How about moving this to `[tikv-client]` section?

https://github.com/pingcap/tidb/blob/5c5aa104d9faea5d87634843680496454fac2569/config/config.toml.example#L262

After all, this parameter affects TiKV only.

Reply:
Make sense, Done.

Reply:
Addressed!